src/faebryk/core/parameter.py

```diff
@@ -514,6 +515,25 @@ def __preinit__(self):
                 continue
             self.operates_on.connect(op.operated_on)
 
+    @once
```

Comment:
`once` works because Expressions are "pretty much" immutable (or should be treated as such"

Reply:
Better to have it defined in the class scope regardless to improve testablity